lib/httparrot/request_store.ex

```diff
@@ -0,0 +1,20 @@
+defmodule HTTParrot.RequestStore do
+
```

Comment:
We could be we could also think that this is tested by the HTTPs tests and not write any as you wish.